client/docs/cli.md

```diff
@@ -0,0 +1,184 @@
+# CLI
+
+The Featureform CLI allows you to describe, list, and monitor your resources.
+
+## Apply
+
+The **apply** command submits resource definitions to the Featureform instance.&#x20;
+
+The argument can either be a path to a local file or the url of a hosted file. Multiple files can be included at a time.
+
+```shell
+featureform apply --host $FEATUREFORM_HOST --cert $FEATUREFORM_CERT <definitions.py>
```

Comment:
Should we mentioned that these can be exported as environment variables as well?